module/ColorJudge.h

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file ColorJudge.h
+ * @brief 色識別クラス
+ * @author hiroto0927, miyashita64
+ */
+
+#ifndef COLOR_JUDGE_H
+#define COLOR_JUDGE_H
+
+#include <algorithm>
+#include "Measurer.h"
+
+enum class COLOR : int {
+  NONE = 0,
+  BLACK = 1,
+  BLUE = 2,
+  GREEN = 3,
+  YELLOW = 4,
+  RED = 5,
+  WHITE = 6,
+};
+
+typedef struct {
+  int hue;         // 色相(0-360)
+  int saturation;  // 彩度(0-100)
+  int value;       // 明度(0-255)
+} hsv_raw_t;
```

Comment:
```suggestion
struct Hsv{
  int hue;         // 色相(0-360)
  int saturation;  // 彩度(0-100)
  int value;       // 明度(0-255)
};
```
C言語の構造体の書き方だね。
C++だとこういう風に書けるよ。実は、C++においてclassとstructはほとんど同じだから、何となくクラスっぽいものはclassで、構造体っぽいものはstructで書いてるらしい。
https://qiita.com/pink_bangbi/items/e25302df7f4ba7fa0db4

あと、型名は単純に`Hsv`でいいんじゃないかな〜
rgb_raw_tはC言語用のEV3APIで定義されているからこんな名前になっているけど、むしろこっちを`Rgb`っていう名前にしたい。 (しないけど)